src/legacy/core_plugins/data/public/search/aggs/buckets/terms.ts

```diff
@@ -58,17 +56,6 @@ export const termsAggFilter = [
   '!sum_bucket',
 ];
 
-const [orderAggSchema] = new Schemas([
-  {
-    group: AggGroupNames.None,
-    name: 'orderAgg',
-    // This string is never visible to the user so it doesn't need to be translated
-    title: 'Order Agg',
-    hideCustomLabel: true,
```

Comment:
The custom label should not be presented in terms -> order agg 

![image](https://user-images.githubusercontent.com/31325372/75981609-ea5a3400-5ef5-11ea-8bc4-c036513c2a78.png)